src/components/App/index.vue

```diff
@@ -178,6 +178,7 @@ export default {
       SessionService.currentSession.data = sessionData;
 
       this.$store.dispatch("user/updateSession", sessionData);
+      this.$store.dispatch("user/sessionConnected");
```

Comment:
is this something we want to run every time session is changed? not super familiar with sockets, but might make more english sense if we renamed this event to `user/sessionChanged` since this event is now going to be called more than just when the session is "connected" initially

Reply:
I initially wanted to ensure that the system doesn't assume that a connection has been established until the server acknowledges that the session has actually been _rejoined_, not merely connected through the socket. However, right now I'm working on a solution that obviates the need to rejoin the session after a connection failure, and so I've moved this dispatch into the `socket.connect()` handler in `SessionView/index.vue`.